src/encoded/tests/data/inserts/higlass_view_config.json

```diff
@@ -193,7 +188,7 @@
                         -166316769.39873636,
                         3257874853.005984
                     ],
-                    "layout": {
+                    "layout": { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5006697)